src/core/dbus-execute.c

```diff
@@ -1229,6 +1229,7 @@ const sd_bus_vtable bus_exec_vtable[] = {
         SD_BUS_PROPERTY("LogRateLimitIntervalUSec", "t", bus_property_get_usec, offsetof(ExecContext, log_ratelimit_interval_usec), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("LogRateLimitBurst", "u", bus_property_get_unsigned, offsetof(ExecContext, log_ratelimit_burst), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("LogExtraFields", "aay", property_get_log_extra_fields, 0, SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("LogFilterRegex", "s", NULL, offsetof(ExecContext, log_filter_regex), SD_BUS_VTABLE_PROPERTY_CONST),
```

Comment:
Not sure, but is it not necessary to be in `dbus-cgroup.c`?

Reply:
`LogFilterRegex` is part of `ExecContext`, so I assumed this should be in `dbus-execute.c`, like other `ExecContext` properties?